pydicom/tests/test_uid.py

```diff
@@ -1,93 +1,22 @@
-# test_uid.py
 """Test suite for uid.py"""
-# Copyright (c) 2008-2012 Darcy Mason
-# This file is part of pydicom, released under a modified MIT license.
-#    See the file LICENSE included with this distribution, also
-#    available at https://github.com/pydicom/pydicom
-
```

Comment:
I agree the code looks cleaner, but from personal experience I've had parts of my code swapped and moved into other people's repos (under their different license) and there is no indication of the original license.